### ISSUE TITLE
[BUG] Exporting data from the existing replicas has issues in 1.6.0

### ISSUE THREAD

**Issue by manju-rn** [longhorn/longhorn#8094]
## Describe the bug
As per this documentation https://longhorn.io/docs/1.6.0/advanced-resources/data-recovery/export-from-replica/  Data from the replicas can be extracted and loaded as a device. However, in the latest documentation of 1.6.0, during the execution of Step 5, it results in error, with longhornio/longhorn-engine:v1.6.0

![image](https://github.com/longhorn/longhorn/assets/26291847/4a9f8788-3eca-45c8-b73d-0b00393b7317)

**Error:**
`longhorn-instance-manager: error while loading shared libraries: libqcow.so.1: cannot open shared object file: No such file or directory`

**Resolution (Temporary)**
However, by using longhornio/longhorn-engine:v1.5.4, it works fine. So believe some libs are missing in longhornio/longhorn-engine:v1.6.0

## To Reproduce

Follow section 5 in https://longhorn.io/docs/1.6.0/advanced-resources/data-recovery/export-from-replica/  to extract the data from the replicas.

Further details are in https://github.com/longhorn/longhorn/discussions/8092

**Comment by derekbit** [comment]
cc @ChanYiLin

**Comment by derekbit** [comment]
#613  cc @c3y1huang

**Comment by c3y1huang** [comment]
> #613 cc @c3y1huang

https://github.com/longhorn/longhorn/issues/7927

**Comment by chriscchien** [comment]
Verified **pass** on longhorn master(longhorn-engine `182b37`)

Follow the [instruction](https://longhorn.io/docs/1.6.0/advanced-resources/data-recovery/export-from-replica/) with longhorn master engine image, replica can be exported to a block device `/dev/longhorn/pvc-850fb12d-a7c3-4c9d-9b76-f10fa943241c` by below command and after mount the block device, the data is correct.

```bash
> docker run -v /dev:/host/dev -v /proc:/host/proc -v /var/lib/longhorn/replicas/pvc-850fb12d-a7c3-4c9d-9b76-f10fa943241c-b6b2dbb1:/volume --privileged longhornio/longhorn-engine:master launch-simple-longhorn pvc-850fb12d-a7c3-4c9d-9b76-f10fa943241c 2147483648
```

**Comment by amaze-escape** [comment]
@chriscchien Thanks for fixing the issue. Is there a way I can try this fix? Will it be published to 1.6.1-rc3?
I've also tried to use 1.5.4 as suggested by @manju-rn . But I get an iSCSI error:
```sh
[pvc-16735494-c40d-43fc-b697-5c2dce53dbfa-e] level=error msg="Failed to init frontend" ... error="device pvc-16735494-c40d-43fc-b697-5c2dce53dbfa: failed to stop iSCSI device: failed to logout target: failed to execute: nsenter [--mount=/host/proc/1969951/ns/mnt --net=/host/proc/1969951/ns/net iscsiadm --version], output , stderr nsenter: failed to execute iscsiadm: No such file or directory\n: exit status 127"
```
So I'm not sure how to proceed right now.
